R/create_credentials.R

```diff
@@ -120,6 +126,39 @@ create_smtp_creds_key <- function(id,
   # Determine whether the keyring package can be used
   validate_keyring_capable()
 
+  # Stop function if `id` provided is not of the right type
+  if (!(inherits(id, "character") | inherits(id, "numeric") | inherits(id, "integer"))) {
```

Comment:
Use `||` instead of `|`

Reply:
Now changed. Thanks for the explanations earlier.